br/backup-and-restore-faq.md

```diff
@@ -44,57 +44,75 @@ aliases: ['/docs-cn/dev/br/backup-and-restore-faq/']
 >
 > 因此，最好在恢复前提前检查权限。
 
-## BR 遇到错误信息 `Io(Os...)`，该如何处理？
+如何检查权限:
+
+1.使用Linux原生进程查询命令：
+
+{{< copyable "shell-regular" >}}
+
+```bash
+ps aux | grep tikv
+```
+
+2.使用tiup命令查询集群启动信息:
+
+{{< copyable "shell-regular" >}}
+
+```bash
+tiup cluster list
+```
+
+## Q5: BR 遇到错误信息 `Io(Os...)`，该如何处理？
 
 这类问题几乎都是 TiKV 在写盘的时候遇到的系统调用错误。例如遇到 `Io(Os { code: 13, kind: PermissionDenied...})` 或者 `Io(Os { code: 2, kind: NotFound...})` 这类错误信息，首先检查备份目录的挂载方式和文件系统，试试看备份到其它文件夹或者其它硬盘。
 
 目前已知备份到 samba 搭建的网盘时可能会遇到 `Code: 22(invalid argument)` 错误。
 
-## BR 遇到错误信息 `rpc error: code = Unavailable desc =...`，该如何处理？
+## Q6: BR 遇到错误信息 `rpc error: code = Unavailable desc =...`，该如何处理？
 
 该问题一般是因为使用 BR 恢复数据的时候，恢复集群的性能不足导致的。可以从恢复集群的监控或者 TiKV 日志来辅助确认。
 
 要解决这类问题，可以尝试扩大集群资源，以及调小恢复时的并发度 (concurrency)，打开限速 (ratelimit) 设置。
 
-## 使用 local storage 的时候，BR 备份的文件会存在哪里？
+## Q7: 使用 local storage 的时候，BR 备份的文件会存在哪里？
 
 在使用 local storage 的时候，会在运行 BR 的节点生成 `backupmeta`，在各个 Region 的 Leader 节点生成备份文件。
 
-## 备份数据有多大，备份会有副本吗？
+## Q8: 备份数据有多大，备份会有副本吗？
 
 备份的时候仅仅在每个 Region 的 Leader 处生成该 Region 的备份文件。因此备份的大小等于数据大小，不会有多余的副本数据。所以最终的总大小大约是 TiKV 数据总量除以副本数。
 
 但是假如想要从本地恢复数据，因为每个 TiKV 都必须要能访问到所有备份文件，在最终恢复的时候会有等同于恢复时 TiKV 节点数量的副本。
 
-## BR 恢复到 TiCDC / Drainer 的上游集群时，要注意些什么？
+## Q9: BR 恢复到 TiCDC / Drainer 的上游集群时，要注意些什么？
 
 + **BR 恢复的数据无法被同步到下游**，因为 BR 直接导入 SST 文件，而下游集群目前没有办法获得上游的 SST 文件。
 
 + 在 4.0.3 版本之前，BR 恢复时产生的 DDL jobs 还可能会让 TiCDC / Drainer 执行异常的 DDL。所以，如果一定要在 TiCDC / Drainer 的上游集群执行恢复，请将 BR 恢复的所有表加入 TiCDC / Drainer 的阻止名单。
 
 TiCDC 可以通过配置项中的 [`filter.rules`](https://github.com/pingcap/ticdc/blob/7c3c2336f98153326912f3cf6ea2fbb7bcc4a20c/cmd/changefeed.toml#L16) 项完成，Drainer 则可以通过 [`syncer.ignore-table`](/tidb-binlog/tidb-binlog-configuration-file.md#ignore-table) 完成。
 
-## BR 会备份表的 `SHARD_ROW_ID_BITS` 和 `PRE_SPLIT_REGIONS` 信息吗？恢复出来的表会有多个 Region 吗？
+## Q10: BR 会备份表的 `SHARD_ROW_ID_BITS` 和 `PRE_SPLIT_REGIONS` 信息吗？恢复出来的表会有多个 Region 吗？
 
 会的，BR 会备份表的 [`SHARD_ROW_ID_BITS` 和 `PRE_SPLIT_REGIONS`](/sql-statements/sql-statement-split-region.md#pre_split_regions) 信息，并恢复成多个 Region。
 
-## 使用 BR 恢复备份数据后，SQL 查询报错 `region is unavailable`
+## Q11: 使用 BR 恢复备份数据后，SQL 查询报错 `region is unavailable`
 
 如果 BR 备份的集群有 TiFlash，恢复时会将 TiFlash 信息存进 `TableInfo`。此时如果恢复的集群没有 TiFlash，则会报该错误。计划在未来版本中修复该错误。
 
-## BR 是否支持就地 (in-place) 全量恢复某个历史备份？
+## Q12: BR 是否支持就地 (in-place) 全量恢复某个历史备份？
```

Comment:
I don't think it is good for doc maintenance. When one FAQ is removed, all FAQ titles must be updated, which might lead to problems.

Reply:
Got it, Remove number.